results_dashboard.py

```diff
@@ -1,24 +1,31 @@
-from app import create_app, db
-from werkzeug.middleware.profiler import ProfilerMiddleware
-# from flask_migrate import Migrate, upgrade
+from app import create_app
+from werkzeug.contrib.profiler import ProfilerMiddleware
 import os
-from dotenv import load_dotenv
+import glob
 
+from app.models.sqlalchemy import DataStore
 
-dotenv_path = os.path.join(os.path.dirname(__file__), '.env')
-if os.path.exists(dotenv_path):
-    load_dotenv(dotenv_path)
```

Comment:
Return back, dotenv is important for production to put passwords and and even your local db passwords.